contentcuration/contentcuration/dev_settings.py

```diff
@@ -11,27 +11,27 @@
 SITE_ID =2
 logging.basicConfig(level='DEBUG')
 
-INSTALLED_APPS += ('debug_toolbar',)
-
-MIDDLEWARE_CLASSES += ('debug_toolbar.middleware.DebugToolbarMiddleware',)
-
-DEBUG_TOOLBAR_CONFIG = {
-	"SHOW_TOOLBAR_CALLBACK": lambda x: True,
-}
-
-
-DEBUG_TOOLBAR_PANELS = [
-    'debug_toolbar.panels.versions.VersionsPanel',
-    'debug_toolbar.panels.timer.TimerPanel',
-    'debug_toolbar.panels.settings.SettingsPanel',
-    'debug_toolbar.panels.headers.HeadersPanel',
-    'debug_toolbar.panels.request.RequestPanel',
-    'debug_toolbar.panels.sql.SQLPanel',
-    'debug_toolbar.panels.staticfiles.StaticFilesPanel',
-    'debug_toolbar.panels.templates.TemplatesPanel',
-    'debug_toolbar.panels.cache.CachePanel',
-    'debug_toolbar.panels.signals.SignalsPanel',
-    'debug_toolbar.panels.logging.LoggingPanel',
-    'debug_toolbar.panels.redirects.RedirectsPanel',
-    'debug_toolbar.panels.profiling.ProfilingPanel',
-]
```

Comment:
If you're usually not wanting it to show, you can update the `SHOW_TOOLBAR_CALLBACK` function to something that requires you to put `?debug` in the URL or something, too, in order for it to show.